src/ruleset/v2/ruleset.ts

```diff
@@ -238,6 +239,7 @@ export const v2SchemasRuleset = (parser: Parser) => {
           },
         },
       },
+      'asyncapi2-message-examples-custom-format': asyncApi2MessageExamplesParserRule(parser),
```

Comment:
Added the rule in but might not want to do this yet. If not, need a pointer on how to get the tests to run without being in the ruleset